sys/newlib/syscalls.c

```diff
@@ -71,8 +72,8 @@ void _fini(void)
  */
 void _exit(int n)
 {
-    printf("#! exit %i: resetting\n", n);
-    reboot(n);
+    LOG_INFO("#! exit %i: resetting\n", n);
```

Comment:
Hm, what happens if you're logging over UDP or on a memory card, for instance? I think if we use logging here, we should make sure that the reboot event actually gets logged.